### ISSUE TITLE
missing "file" field on export

### ISSUE THREAD

**Issue by brynpickering** [retorquere/zotero-better-bibtex#671]
Hi, I recently refreshed firefox after a few years of no issues working with Zotero, Zotfile, and Better Bib(La)Tex. On refresh, my exported .bib files don't include the 'file' field, which should point towards PDFs in my locally stored Zotero library. On exporting with Zotero's BibTex, they are there and they were working fine with Better Bib(La)Tex before the refresh. Not sure if it is an actual bug or a setting that I just can't find.

I filed a report either way: WJ7JRRCG

Issue occurs on export with either Better BibTex or Better BibLaTex.

**Comment by retorquere** [comment]
Nothing of interest in the current log. Can you enable extended debugging and submit a new one?

**Comment by brynpickering** [comment]
Updated log: A3X4RJ6B

**Comment by retorquere** [comment]
Nothing out of the ordinary. Is this an automatic export or a manual export?

**Comment by brynpickering** [comment]
Looks to be an issue in Zotfile, not with you! I'll close this issue, but for the record: I had to apply 'rename attachments' via 'Manage attachments' in zotfile for all my library entries. Better bibtex was then able see the files on my local device and apply the 'file' field accordingly.

**Comment by retorquere** [comment]
That is really strange. If zotfile is deactivated, does bbt work as expected?

**Comment by brynpickering** [comment]
It does, although obviously any new library entry stores its PDF in the Zotero storage, not in the custom folder structure that Zotfile would give me. It points to that no problem. The issue just occurred when I reconnected my Zotero plugin to my offline library, it lost all knowledge of there being local files. New entries now have a 'file' field automatically, as was the case before I refreshed Firefox
